integration-tests/contracts/ethereum_vrf_contracts.go

```diff
@@ -12,19 +12,20 @@ import (
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/rs/zerolog/log"
 
+	"github.com/smartcontractkit/seth"
+
 	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
 	"github.com/smartcontractkit/chainlink/integration-tests/wrappers"
 	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/batch_blockhash_store"
 	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/blockhash_store"
 	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/solidity_vrf_consumer_interface"
-	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/solidity_vrf_coordinator_interface"
 	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/solidity_vrf_wrapper"
 )
 
-// LegacyEthereumBatchBlockhashStore represents BatchBlockhashStore contract
-type LegacyEthereumBatchBlockhashStore struct {
-	address             *common.Address
-	client              blockchain.EVMClient
+// EthereumBatchBlockhashStore represents BatchBlockhashStore contract
+type EthereumBatchBlockhashStore struct {
+	address             common.Address
+	client              *seth.Client
 	batchBlockhashStore *batch_blockhash_store.BatchBlockhashStore
 }
```

Comment:
I am a bit confused here. Why some of old contract wrappers are modified in situ and another were renamed to `Legacy*` and created in `integration-tests/contracts/ethereum_vrf_contracts_seth.go`?

Reply:
@Tofel let me remove the `Legacy*` contracts I added in this PR

Reply:
Removed in 212c2a3fa2bd058d5ad6b8150ceef5ddde3cd8cc